### PR TITLE
Adds a draft About page to site

### PR DESCRIPTION
**Pull request by sdellis**
This is the first commit for adding an About page. Additional information and examples are necessary before merging.

### CODE REVIEW

**Comment by sdellis** [comment]
The heading CSS on the site are a little odd. H1 headings seem smaller than H2. 
Screen Shot 2022-07-25 at 4 56 15 PM

File: app/controllers/about_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class AboutController < ApplicationController
+  def index
```
Comment (eliotjordan) [inline app/controllers/about_controller.rb:4]: Try `def index; end`

Comment (sdellis) [reply app/controllers/about_controller.rb:4]: That worked! Thanks, @eliotjordan !

File: app/views/about/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  Genealogists, local history buffs, city planners, architects, and countless others have
+  been drawn to the quality of detail found in them. </p>
+
+<p>
+  <%=image_tag("about/pulmap-sanborn.jpg", alt: "Example of a Sandborn map of Princeton, Mercer County, New Jersey", width: "800px")%>
+</p>
+
+<h3>Scanned Maps</h3>
+<p>
+Scanned Maps are print maps that have been digitized and combined with geographic information, allowing for accurate positioning of the map on an interactive, modern globe. Many of the Scanned Maps available in the Digital Map and Geospatial Data Portal are historic, allowing researchers to compare past geographic boundaries and names with their modern counterparts. Princeton University Library has a strong commitment to scanning our print maps collection, so scanned maps are added to the portal on a regular basis.
```
Comment (eliotjordan) [inline app/views/about/index.html.erb:37]: `and combined with geographic information, allowing for accurate positioning of the map on an interactive, modern globe` is misleading. I think it's fine to just say "digitized print maps".

File: app/views/about/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+[Example Scanned Map Here.]
+</p>
+
+<h3>Vector Data</h3>
+<p>
+Vector graphics are stored via mathematical calculations from one point to another that form smooth lines and shapes. They can be resized to any dimension without losing clarity. Vector Data is spatially referenced data which is represented by three types of marks: point, line (or arc), and polygon data.
```
Comment (eliotjordan) [inline app/views/about/index.html.erb:44]: I'd ask Wangyal for a definition here.

File: app/views/about/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+Point Data is most commonly used to represent non-adjacent features and to represent discrete data points. Examples would be bus stops, schools, city names, and other points of interest.
+Line Data is ...
+Polygon Data is ...
+[Example Images of the three above Vector data types.]
+</p>
+
+<h3>Raster Data</h3>
+<p>
+Raster images are made of tiny squares called pixels. Raster data (also known as grid data) represents “surfaces”. Raster data is grid-based, often by pixel, and each unit expresses a data value that is represented visually. Raster Data often includes topographies as well as aerial and satellite imagery. There are two types of raster data: continuous and discrete. An example of discrete raster data is population density. Continuous data examples are temperature and elevation measurements. There are also three types of raster datasets: thematic data, spectral data, and pictures (imagery).
```
Comment (eliotjordan) [inline app/views/about/index.html.erb:54]: Same as above. Ask Wangyal.

File: app/views/about/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+</p><p>
+Thematic Data can be used for Digital Elevation Model (DEM). Each cell represents a 30m pixel size with an elevation value assigned to that cell.
+Spectral Data can be used for spatial hydrology modeling such as extracting watersheds and flow lines in a geographic area. Spectral data is often used in aerial or satellite imagery which, for example, can be used to derive vegetation geologic information by classifying the spectral signatures of each type of feature.
+Image Aerial and satellite imagery is one type of raster data. Raster Image file types include BMP, TIFF, GIF, and JPEG.
+[Example Image for each of the Raster data types here.]
+</p><p>
+Raster images accompanied by a second file known as a world file. The world file has the same name as the raster image file but has a different extension. The world file is a text file that contains the map projection information needed to properly georeference the raster image.
```
Comment (eliotjordan) [inline app/views/about/index.html.erb:61]: We don't use world files. The geotiff headers contain the spatial reference information.

File: app/views/about/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+</p>
+
+<h3>Subject Areas</h3>
+
+<p>The maps and data in this collection may be used for many areas of scholarship, but highlights include:</p>
+<ul>
+  <li>Geology</li>
```
Comment (eliotjordan) [inline app/views/about/index.html.erb:68]: Should these be links?

File: app/views/about/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+</p>
+<h2>Types of Maps found in the Portal</h2>
+
+<p>Please note that this Portal represents only a subsection of the maps that
+  Princeton University Library has in its collection. [maybe add some text about how/where to
+  locate the maps that are not discoverable in this portal?]
+
+<h3>Sanborn Maps</h3>
+
+<p>The Sanborn Map Company is the oldest mapping company in the United States.
```
Comment (eliotjordan) [inline app/views/about/index.html.erb:21]: This is pretty good. Might need a few tweaks here and there. I'd get Kim's opinion on this and, maybe, whoever is responsible for Sanborns in Special Collections these days. Emma Sarconi?

Comment (sdellis) [reply app/views/about/index.html.erb:21]: @eliotjordan sounds good. @kelea99 any thoughts on this text about Sanborn maps? I got this info from the Special Collections website, but always good to check to make sure it's up to date.

File: app/views/about/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,91 @@
+<% @page_title = t('blacklight.about.page_title', :application_name => application_name) %>
+
+<%= render 'shared/start_over_row' %>
+
+<div id="content" class="col-lg-12">
+<h1 class='page-heading'><%= t('blacklight.about.title') %></h1>
+<p>
+  The Princeton University Library’s Digital Maps and Geospatial Data website
+  allows researchers to discover and access online maps and geospatial data in
+  the Princeton Maps Collection, one of the largest cartographic collections
```
[the remark this answers is not in the document]
Comment (sdellis) [reply app/views/about/index.html.erb:10]: According to this page (first sentence), it's true: https://library.princeton.edu/collections/pumagic/collections

File: app/views/about/index.html.erb (hunk 2 above)
Comment (sdellis) [reply app/views/about/index.html.erb:37]: I was going for something that would tie these maps to GIS and Geospatial discovery. "Digitized print maps" that have no GIS information would not be included in this portal, or would they?

File: app/views/about/index.html.erb (hunk 5 above)
Comment (sdellis) [reply app/views/about/index.html.erb:61]: Ok, gotcha. Then do I understand correctly that we ONLY have geotiff raster files in Pulmap? (Obviously TIFF, BMP, GIF, or JPEG would not have geotiff headers, so they would not have any spatial reference info, but just want to confirm.)

File: app/views/about/index.html.erb (hunk 6 above)
Comment (sdellis) [reply app/views/about/index.html.erb:68]: I don't think so. But I'm open to ideas. Do you mean links to subject headings?

File: app/views/about/index.html.erb (hunk 7 above)
Comment (kelea99) [reply app/views/about/index.html.erb:21]: This sounds good to me.

Comment (kelea99) [reply app/views/about/index.html.erb:21]: @sdellis Emma is your point person

**Comment by hackartisan** [comment]
Do we want the "start over" button to show on this page? It's a little confusing to me.

**Comment by sdellis** [comment]
@hackartisan I was wondering the same thing, but since the Contact Us page had it, I figured we should at least be consistent. I'm happy to remove it.

**Review by eliotjordan — CHANGES_REQUESTED** [review]
Could the example images be reduced in size? Some of them are in the 3-7 MB range which seems too large for an about page.

**Comment by eliotjordan** [comment]
Closing. If we want to do this in the future, we can.

**Pull request closed without merging**